src/build_matrix_A.py

```diff
@@ -38,23 +35,23 @@ def build_matrix_A(material_property_library, mesh, time):
     volume = 4/3*np.pi*(mesh[0][1][0]+DR[0]/2)**3
     
     
-    A[0, 0] = k[0]*Dt*4*np.pi/(rho[0]*c[0])/(DR[0]*volume)*(DR[0]/2)**2
+    A[0, 0] = k[0]*4*np.pi/(rho[0]*c[0])/(DR[0]*volume)*(DR[0]/2)**2
     A[0, 1] = -A[0,0]
 	
 
     volume = 4/3*np.pi*(mesh[0][1][1]+DR[0]/2)**3-(mesh[0][1][1]-DR[0]/2)**3
     
-    A[1, 0] = -k[0]*Dt*4*np.pi/(rho[0]*c[0])/(DR[0]*volume)*(mesh[0][1][0]+DR[0]/2)**2
-    A[1, 2] = -k[0]*Dt*4*np.pi/(rho[0]*c[0])/volume*(1/(1/mesh[0][1][1]-1/mesh[0][1][2]))
+    A[1, 0] = -k[0]*4*np.pi/(rho[0]*c[0])/(DR[0]*volume)*(mesh[0][1][0]+DR[0]/2)**2
+    A[1, 2] = -k[0]*4*np.pi/(rho[0]*c[0])/volume*(1/(1/mesh[0][1][1]-1/mesh[0][1][2]))
     A[1, 1] = -(A[1,0] +A[1,2])
 
 	
     for i in range(2, len(mesh[0][1])-1): #Length set to fueled node length
         volume = 4/3*np.pi*((mesh[0][1][i]+DR[0]/2)**3-(mesh[0][1][i]-DR[0]/2)**3)
-        A[i, i] = k[0]*Dt*4*np.pi/(rho[0]*c[0])/volume*(1/(1/mesh[0][1][i]-1/mesh[0][1][i+1]) +
+        A[i, i] = k[0]*4*np.pi/(rho[0]*c[0])/volume*(1/(1/mesh[0][1][i]-1/mesh[0][1][i+1]) +
```

Comment:
still a fan of `A[i,i] = -A[i, i+1] - A[i,i-1]` for this (I think that's right...)